File: 시저_암호.py
def solution(s, n):
    result = ""
    for i in range(len(s)):
        if s[i].isalpha():
            if s[i].isupper():
                tmp = ord(s[i]) - ord('A')
                tmp += n
                tmp %= 26
                result += chr(tmp + ord('A'))
            else:
                tmp = ord(s[i]) - ord('a')
                tmp += n
                tmp %= 26
                result += chr(tmp + ord('a'))
        else:
            result += s[i]
            
    return result

File: 시저_암호.cpp
#include <iostream>
using namespace std;

string solution(string s, int n) {
	for (int i = 0; i < s.length(); i++) {
		if (isalpha(s[i])) {
			if (isupper(s[i])) {
				int tmp = s[i] - 'A';
				tmp += n;
				tmp %= 26;
				s[i] = tmp + 'A';
			}
			else {
				int tmp = s[i] - 'a';
				tmp += n;
				tmp %= 26;
				s[i] = tmp + 'a';
			}
		}
	}

	return s;
}
